data_source.go

```diff
@@ -73,6 +91,7 @@ type NodeInfo struct {
 	Role                string             `json:"role"`
 	Xlog                *XlogInfo          `json:"xlog"`
 	Replication         []*ReplicationInfo `json:"replication"`
+	PgCurrentWalLsn     string             `json:"pg_current_wal_lsn"`
```

Comment:
TODO: remove this field out of NodeInfo struct. calculate the byte lag using `pg_wal_lsn_diff